ctrd/watch.go

```diff
@@ -16,24 +16,13 @@ type Message struct {
 	exitCode uint32
 	exitTime time.Time
 	err      error
-	startErr error
-}
-
-// StartError returns the error of start exec process or container.
-func (m *Message) StartError() error {
-	return m.startErr
 }
 
 // Error returns the error contained in Message.
 func (m *Message) Error() error {
```

Comment:
Sometimes I see code `m.Error().Error()`. It is some kind of weird.
Could we change this function like `RawError()` to distinguish from Error() which is well-known as a interface function.